.github/workflows/up_ex_doc_version.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: up_ex_doc_version
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight
+  workflow_dispatch:
+  
+
+jobs:
+  test:
+    name: Update ex_doc version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get current version
```

Comment:
```suggestion
      - uses: erlef/setup-beam@v1
        with:
          otp-version: 24
          elixir-version: 1.13
      - name: Get current version
```

Reply:
Without this `mix` isn't available (it's used below).

Also, FYI, I picked the same Erlang/Elixir versions as from the other workflow.